spec/index.bs

```diff
@@ -410,6 +410,15 @@ The [=subresources=] of textures included in the views provided to
 are considered to have {{GPUTextureUsage/OUTPUT_ATTACHMENT}}
 for the [=usage scope=] of this render pass.
 
+The <dfn dfn>physical size</dfn> of a {{GPUTexture}} [=subresource=] is the dimension of the {{GPUTexture}}
+[=subresource=] in pixels that includes the possible extra paddings to form complete [=texel blocks=] in the
+[=subresource=]. The extra paddings only exists in a compressed texture mipmap whose {{GPUTexture/[[textureSize]]}}
+at [=mipmap level=] 0 are multiple of [=texel block size=], but its subdivided levels are not. For example,
```

Comment:
Instead of "For examples," can we use bikesheds formatting like:
```
Example: xxx
```

Reply:
Done